app/uk/gov/hmrc/soletraderidentificationfrontend/services/AuditService.scala

```diff
@@ -74,26 +74,23 @@ class AuditService @Inject()(auditConnector: AuditConnector, soleTraderIdentific
 
   def auditSoleTraderJourney(journeyId: String)(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[Unit] = {
     for {
-      optFullName <- soleTraderIdentificationService.retrieveFullName(journeyId)
-      optDateOfBirth <- soleTraderIdentificationService.retrieveDateOfBirth(journeyId)
+      optSoleTraderRecord <- soleTraderIdentificationService.retrieveSoleTraderDetails(journeyId)
       optNino <- soleTraderIdentificationService.retrieveNino(journeyId)
```

Comment:
The nino also gets pulled back in the optSoleTraderRecord so could we remove this extra nino call as well maybe?

Reply:
fixed